---
title: >-
  precisionFDA Truth Challenge V2 Calling variants from short- and long-reads in
  difficult-to-map regions
created: '2021-03-11T17:23:48.850246'
modified: '2021-03-11T17:23:48.850255'
state: active
type: dataset
tags:
  - Benchmarking
  - Bioinformatics
  - Genome In A Bottle
  - Precisionfda
  - Variant Calling
  - Whole Genome Sequencing
groups: []
csv_url: >-
  https://data.nist.gov/od/ds/ark:/88434/mds2-2336/benchmarking_results/YUI27/YUI27_HG004.extended.csv
json_url: ''
layout: post

---
The precisionFDA Truth Challenge V2 aimed to assess the state-of-the-art of variant calling in difficult-to-map regions and the Major Histocompatibility Complex (MHC). Starting with FASTQ files, 20 challenge participants applied their variant calling pipelines and submitted 64 variant callsets for one or more sequencing technologies (~35X Illumina, ~35X PacBio HiFi, and ~50X Oxford Nanopore Technologies). Submissions were evaluated following best practices for benchmarking small variants with the new GIAB benchmark sets and genome stratifications. Challenge submissions included a number of innovative methods for all three technologies, with graph-based and machine-learning methods scoring best for short-read and long-read datasets, respectively. New methods out-performed the 2016 Truth Challenge winners, and new machine-learning approaches combining multiple sequencing technologies performed particularly well. Recent developments in sequencing and variant calling have enabled benchmarking variants in challenging genomic regions, paving the way for the identification of previously unknown clinically relevant variants. This dataset includes the fastq files provided to participants, the submitted variant callset as vcfs, and the benchmarking results, along with challenge submission metadata. 
